Mathlib/Analysis/Calculus/IteratedDeriv/Lemmas.lean

```diff
@@ -108,14 +141,12 @@ theorem iteratedDeriv_const_smul {n : ℕ} {f : 𝕜 → F} (h : ContDiff 𝕜 n
       ← Function.comp_def, deriv.scomp x h₀ (differentiableAt_id'.const_mul _),
       deriv_const_mul _ differentiableAt_id', deriv_id'', smul_smul, mul_one, pow_succ]
 
+/-- Note: this is unrelated to `iteratedDerivWithin_const_mul`, where the multiplication works on
+the codomain. -/
 theorem iteratedDeriv_const_mul {n : ℕ} {f : 𝕜 → 𝕜} (h : ContDiff 𝕜 n f) (c : 𝕜) :
```

Comment:
Hmm, maybe this should be called `iteratedDeriv_comp_const_mul`?

Reply:
I'll do this in a followup